Polyhedron/demo/Polyhedron/resources/heat_intensity_shader.f

```diff
@@ -76,9 +76,11 @@
     h = h - floor(h);
     h = (1./(1.+exp(-100.*(h-.55)))) + (1./(1.+exp(-100.*(-h+.45))));
     h = 1.-h;
-    
-    c = h*vec3(1.,1.,1.) + (1.-h)*c;
-    
+#ifdef HEAT_METHOD_BLACK_LINES
+      c = h*vec3(0.,0.,0.) + (1.-h)*c;
+#else
+      c = h*vec3(1.,1.,1.) + (1.-h)*c;
+#endif
```

Comment:
That will not work, because it is an OpenGL shader. @maxGimeno, can you review and suggest something that works, for Andreas?

Reply:
From [the glsl wiki page](https://www.khronos.org/opengl/wiki/Core_Language_(GLSL)#Preprocessor_directives):
 All of the keywords beginning with # are preprocessor directives, much like with C, although #line is different. GLSL provides most of the standard C set of preprocessor directives (#define, #if, etc), in addition to the ones listed above. The most notable omission is #include.
So my guess it that it should work...

Reply:
That I know. But have a look at the commit https://github.com/CGAL/cgal/pull/3452/commits/0e6d896466bd0de720bfdb16d7aadeca564bffd7. I do not see how the `#define HEAT_METHOD_BLACK_LINES` in the .cpp file can have any effect on the processor conditionals in the shader file.

Reply:
Oh, ok. It makes sense. Why not simply move the #define in the shader then ?